src/main/java/uk/gov/justice/digital/delius/transformers/StaffTransformer.java

```diff
@@ -40,10 +40,13 @@ static Human humanOf(Staff staff) {
                 .forenames(combinedMiddleNamesOf(staff.getForename(), staff.getForname2()))
                 .surname(staff.getSurname()).build();
     }
-    static ContactableHuman contactableHumanOf(Staff staff) {
+    static ContactableHuman contactableHumanOf(Staff staff, Optional<String> email, Optional<String> phoneNumber) {
```

Comment:
You could just make these Strings rather than Optional if that improves code noise